### PR TITLE
Supports component

### PR DESCRIPTION
**Pull request by Fraasi**
This is the 'Support The Coding Train' part of the home page.  
There's some room for improvement on the css-side of things, but other than that this should be done.

### CODE REVIEW

File: src/components/Supports/index.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import patreonImg from '../../assets/images/patreon.svg';
+import merchImg from '../../assets/images/merch-store.png';
+import amazonImg from '../../assets/images/amazon-store.png';
+import Button from '../Button';
+
+export default class Supports extends Component {
+  render() {
+    return (
+      <>
+        <Title>SUPPORT THE CODING TRAIN</Title>
```
Comment (meiamsome) [inline src/components/Supports/index.tsx]: Prefer "Support the Coding Train" - our current font just looks upper case but they're technically different

File: src/components/Supports/index.styles.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+export const MediaWrapper = styled.div`
+  display: flex;
+  ${props => props.theme.media.tablet`
+    flex-wrap: wrap;
+  `}
+`;
+
+export const StyledWrapper = styled.div`
+  font-family: "Open Sans",sans-serif;
```
Comment (meiamsome) [inline src/components/Supports/index.styles.tsx]: Can we add a space after the comma?

Comment (Fraasi) [reply src/components/Supports/index.styles.tsx]: We certainly can.

File: src/components/Supports/index.tsx (hunk 1 above)
Comment (Fraasi) [reply src/components/Supports/index.tsx]: And you never know if we ever change the font sometime down the road.

**Pull request merged**